scripts/build/Dockerfile.openj9-ubuntu

```diff
@@ -1,4 +1,4 @@
-FROM adoptopenjdk/openjdk8-openj9:latest
+FROM eclipse-temurin:8-focal
```

Comment:
If we change it we could move away from short-names to a container name including the registry. Can you change that?

Reply:
You're right, I appreciate your mentioning that.
I've replaced the short-name with `docker.io/library/eclipse-temurin:8-focal` to include the registry.